dev-client/src/util.ts

```diff
@@ -25,3 +25,10 @@ export const removeKeys = (a: any, b: any) => {
     }
   }
 };
+
+export const normalizeText = (text: string) =>
```

Comment:
this is a duplicate of `cleanSensitiveCharacters` in the web client. Please refactor and move to shared client.